src/status_im2/contexts/profile/settings/header/utils.cljs

```diff
@@ -0,0 +1,21 @@
+(ns status-im2.contexts.profile.settings.header.utils
+  (:require [status-im2.constants :as constants]
+            [utils.i18n :as i18n]))
+
+(defn visibility-status-type-data
+  [status]
+  (case (:status-type status)
+    (constants/visibility-status-automatic)
+    {:title (i18n/label :t/status-automatic)
+     :icon  :i/automatic}
+
+    (constants/visibility-status-always-online)
+    {:title (i18n/label :t/online)
+     :icon  :i/online}
+
+    (constants/visibility-status-inactive)
+    {:title (i18n/label :t/offline)
+     :icon  :i/offline}
+
+    {:title (i18n/label :t/offline)
+     :icon  :i/offline}))
```

Comment:
The `def` vars `visibility-status-automatic`, `visibility-status-always-online` and `visibility-status-inactive` are not functions. We should remove the wrapped brackets.

Reply:
@smohamedjavid without brackets i have this error `clj-kondo: Case test symbol is compile time constant and is never evaluated.`
![image](https://github.com/status-im/status-mobile/assets/71308738/34a10464-c2e2-4d9a-a673-6004783c692f)

Reply:
Oh, Apologies. It's the nature of `case`. 
Thanks for checking it 👍